public/literals.json

```diff
@@ -7,5 +7,6 @@
   "webgl_warning": "⚠️ There is a problem with your current configuration (WebGL is disabled or unavailable). The map won't be able to load.",
   "deprecated_layers_warning": "The activity layer $LAYERS used on this workspace is deprecated and may not get updated data, click <a href='http://globalfishingwatch.org/map'>here</a> to get the latest one.",
   "deprecated_layers_warning_plural": "The activity layers $LAYERS used on this workspace are deprecated and they may not get updated data, click <a href='http://globalfishingwatch.org/map'>here</a> to get the latest ones.",
-  "edge_warning": "Map performance might be degraded using Edge. We recommand using another browser such as Firefox or Chrome."
+  "edge_warning": "Map performance might be degraded using Edge. We recommand using another browser such as Firefox or Chrome.",
+  "vessel_warning": "This vessel has comments, please review them"
```

Comment:
@j8seangel where is this key being used? I've search the PR looking for `vessel_warning` and the one in literals.json is the only one that appeared.

Reply:
Sorry, this is my fault and I forgot to add it as the title of the warning icon in the vessel info.

Just to clarify the workflow when a vessel info request has the comment field on the response:
- The warning notification bar will be shown the content of the comment itself automatically.
- Vessel info and the pinned vessel sections will show a warning icon with the above comment as a title that triggers the notification with the comment as a warning.